packages/node_modules/pouchdb-core/src/setup.js

```diff
@@ -127,4 +128,54 @@ PouchDB.fetch = function (url, opts) {
   return fetch(url, opts);
 };
 
+function ActiveTasks() {
+  this.tasks = [];
+}
+
+ActiveTasks.prototype.list = function () {
+  return this.tasks
+};
+
+ActiveTasks.prototype.add = function (task) {
+  const newTask = {
+    name: task.name,
+    total_items: task.total_items,
+    id: uuidv4().substr(0,8),
```

Comment:
we should have a good reason for cursing up a uuid, why not leave this at default length?

Reply:
Oh, just readability while testing, that should have had a comment